ntoskrnl/ex/interlocked.c

```diff
@@ -184,8 +184,8 @@ ExInterlockedRemoveHeadList(
         /* Remove the first entry from the list head */
         ListEntry = RemoveHeadList(ListHead);
 #if DBG
-        ListEntry->Flink = (PLIST_ENTRY)0xBADDD0FF;
-        ListEntry->Blink = (PLIST_ENTRY)0xBADDD0FF;
+        ListEntry->Flink = (PLIST_ENTRY)(ULONG_PTR)0xBADDD0FFBADDD0FFULL;
```

Comment:
Can we have these debug patterns in some defines?

Reply:
I don't see what that would achieve.